sfcasts/assets.md

```diff
@@ -1,96 +1,43 @@
+
+You probably noticed that there is an `addWebpackEncoreEntry` that we can do here. So if I said `'app'` here, that would pull in the CSS and JavaScript from my app entry. *But* that makes things look a little crazy because we don't want all of our front end styles and JavaScript to show up the admin section. We just want to be able to add a *little bit* of new stuff.
+
+Here's what we'll do instead. Inside the `assets/styles/` directory, let's create an entirely new file here called `admin.css`. This will be our CSS solely for configuring the admin section. And just to see if things are working, I'll add a very lovely body background of "lightcyan". Ooh... fancy.
+
+Over in `webpack.config.js`, I'm going to add a second entry for *just* the, admin. But, right now, since I only have a CSS file (I don't need JavaScript), I'll say `.addStyleEntry`... call it `'app'`... and point it to `.assets/styles/admin.css`.
+
+Perfect! Because I just modified my Webpack file, I need to go over to my terminal, find where I'm running Webpack, hit "ctrl + C", and then rerun it.
+
+And... it exploded. You probably saw me make a mistake there. I need to give my entry a unique name, so I'll change `'app'` to `'admin'`, run it again, and... beautiful! And now, in addition to the original stuff, you can see it also dumped an `admin.css` file. So, thanks to this, over in our `DashboardController`, we'll do `->addWebpackEncoreEntry('admin');`. 
+
+Refresh and... it works! Lovely.
+
+If you go to "View Page Source", you can see how this works. There's really nothing special here. The `app.css` file here gives us all of the EasyAdmin styling that we've been enjoying, and *here* is our new `admin.css` file.
+
+At this point, we're dangerous. We can add whatever CSS we want to this new `admin.css` file, and it will override *any* of the EasyAdmin styles. Cool! But EasyAdmin makes it even easier than that!
+
+Inspect the element on the sidebar here. I just want to change this background a little bit. Find the actual element with the `"sidebar"` class. If you look over here...I'll make this a little bit bigger... you can see that we have `.sidebar` `background`, but instead of being set to a color, it's set to this `var(--sidebar-bg)` thing. If you hover over it, you can see that, apparently, it's equal to `#f8fafc`.
+
+If you haven't seen it before, this is a CSS property. It has nothing to do with EasyAdmin or Symfony. In CSS, you can create variables (called "CSS properties") and reference them somewhere else. EasyAdmin has created a `--sidebar-bg` variable, and it's referencing it here. So, instead of trying to override the background of `.sidebar`, we can just override this CSS property and it will have the same effect. How do we do that? I'm gonna have you cheat a little bit. We're going to dig down deep into EasyAdmin itself.
```

Comment:
This one comes up a few times. If you're talking about "sidebar-bg" (or any similar variables) by itself, it technically reads with two leading dashes in the video, even though you don't verbalize them. I added them in hopes of being more accurate, but if you think it would be better to leave them out, let me know and I'll fix it.